src/java.desktop/share/classes/javax/swing/border/EtchedBorder.java

```diff
@@ -24,14 +24,12 @@
  */
 package javax.swing.border;
 
-import java.awt.BasicStroke;
+import com.sun.java.swing.SwingUtilities3;
```

Comment:
In JDK code, the convention is to put `com.*` and `sun.*` packages after the standard public API ones under `java.*` and `javax.*`.